Rebus.Idempotency/Rebus.Idempotency.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <VersionPrefix>2.1.0</VersionPrefix>
+    <VersionPrefix>3.0.0</VersionPrefix>
```

Comment:
Does this include a breaking change? I thought we were going to be backwards compatible so shouldn't we use 2.2.0?

Reply:
Yes, maybe that would be desirable indeed. I'll update it